python/xgbserver/xgbserver/model.py

```diff
@@ -43,7 +44,7 @@ def load(self) -> bool:
     def predict(self, request: Dict) -> Dict:
         try:
             # Use of list as input is deprecated see https://github.com/dmlc/xgboost/pull/3970
```

Comment:
@yuzisun I am curious why this hasn't been fixed before even though there is a comment pointing to the issue in xgboost.

Reply:
@pradithya We are mostly moving to v2 protocol at this point for xgboost and I think it is fixed there https://github.com/SeldonIO/MLServer/blob/739ebbaf92f1c135c789df009f2c386c6ab6f49e/runtimes/xgboost/mlserver_xgboost/xgboost.py#L49.

Reply:
Got it, thanks for clarifying @yuzisun